pennylane/devices/qubit/preprocess.py

```diff
@@ -0,0 +1,220 @@
+# Copyright 2018-2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This module contains functions for preprocessing `QuantumScript`s to ensure
```

Comment:
```suggestion
"""This module contains functions for preprocessing :class:`~.QuantumTape`s to ensure
```
there are a few other spots in the PR that need this update too. Reminder, single-backticks are for code in markdown, double-backticks are for code in Sphinx docs, and you can use the above syntax to hyperlink to a class's doc